proposals/0079-allow_commission_decrease_at_any_time.md

```diff
@@ -0,0 +1,83 @@
+---
+simd: '0079'
+title: Allow Commission Decrease at Any Time
+authors:
+  - Bryan Ischo (bryan@ischo.com)
+category: Standard
+type: Core
+status: Draft
+created: 2023-10-26
+feature: TBD
+---
+
+## Summary
+
+The commission_updates_only_allowed_in_first_half_of_epoch feature disallows
+commission decrease in the second half of epoch. Given that the purpose of this
+feature was to prevent 'rug pulls' which are accomplished by increasing
+commission at the end of epoch and then decreasing commission at the beginning
+of next epoch, disallowing decreases during the second half of epoch is
+unnecessary.
+
+A feature gate must be added to support this SIMD as all validators' vote
+programs must treat commission change instructions the same or else consensus
+will diverge.
+
+## Motivation
+
+Some validator operators may need to decrease commission in order to satisfy
+their own operational criteria.  Speaking personally, when I have an operator
+error that results in reduced stake account rewards for the epoch, I reduce
+commission to 0% to ensure that stake accounts are not disadvantaged by my
+actions. Not being allowed to do this in the second half of an epoch is a real
+problem for me.
```

Comment:
This is extremely useful, thanks for providing the story. As a nit, please clarify who "I" is, so that the use case becomes clear to anyone reading the proposal.

Reply:
I have altered the document to no longer contain first-person wording.  I describe the generalized operational technique now.